### DISCUSSION TITLE
Cannot accept the assignment

### DISCUSSION THREAD

**Discussion by ManzanoHerchelle** [community/community#140750] [GitHub Education]
### Hi everyone,

I just started using my student developer pack here in github and chose to begin my "Intro to github flow lesson". This is what showed up upon clicking it and I am not able to accept the assignment. Please help me figure this out ASAP! Thank you.

<img width="682" alt="Screenshot 2024-10-07 at 12 27 44 PM" src="https://github.com/user-attachments/assets/6cd28933-f6d7-4cbb-a586-c1f2c54841ef">

**Comment by AbdulAhad365** [comment]
in think your application is in process, you need to wait for its completion. once, accepted you can use it

else: u can email to github official
